packages/excalidraw/constants.ts

```diff
@@ -255,6 +255,14 @@ export const EXPORT_SOURCE =
 // time in milliseconds
 export const IMAGE_RENDER_TIMEOUT = 500;
 export const TAP_TWICE_TIMEOUT = 300;
+/**
+ * The time the user has from 2nd pointerdown to following pointerup
+ * before it's not considered a double click.
+ *
+ * Helps prevent cases where you double-click by mistake but then drag/keep
+ * the pointer down for to cancel the double click or do another action.
+ */
```

Comment:
Just a thought: Wouldn't it make more sense to detect pointer displacement with a threshold? Some people with disabilities might not be able to release the button within such a short time (300ms currently).

Reply:
I don't think it's an accessibility concern as the same issue would apply to the double `pointerdown` to begin with. What I'm doing here is about `pointerup`. In other words, if you've already managed to double-click, then the pointer release shouldn't be a problem.